webapp/src/language/index.js

```diff
@@ -5,24 +5,32 @@ import en from './en.json'
 import es from './es.json'
 import enLacchain from './en.lacchain.json'
 import esLacchain from './es.lacchain.json'
-import enTelos from './en.telos.json'
-import esTelos from './es.telos.json'
+import enTelos from './en.telos-testnet.json'
+import esTelos from './es.telos-testnet.json'
 import enJungle from './en.jungle.json'
 import esJungle from './es.jungle.json'
 import enMainnet from './en.mainnet.json'
 import esMainnet from './es.mainnet.json'
+import enProton from './en.proton-testnet.json'
+import esProton from './es.proton-testnet.json'
+import enWax from './en.wax-testnet.json'
+import esWax from './es.wax-testnet.json'
 
 const languajes = {
   es,
   en,
   'es.lacchain': esLacchain,
   'en.lacchain': enLacchain,
-  'es.telos': esTelos,
-  'en.telos': enTelos,
+  'es.telos-testnet': esTelos,
```

Comment:
Teto ets add support for the following networks 

[{
    "label": "EOS Mainnet",
    "value": "https://mainnet.eosio.online"
}, {
    "label": "LACChain EOSIO",
    "value": "https://lacchain.eosio.online"
}, {
    "label": "Proton",
    "value": "https://proton.eosio.online"
}, {
    "label": "WAX",
    "value": "https://wax.eosio.online"
}, {
    "label": "Telos",
    "value": "https://telos-testnet.eosio.online"
}, {
    "label": "Jungle Testnet",
    "value": "https://jungle.eosio.online"
}, {
    "label": "Proton Testnet",
    "value": "https://proton-testnet.eosio.online"
}, {
    "label": "WAX Testnet",
    "value": "https://wax-testnet.eosio.online"
}, {
    "label": "Telos Testnet",
    "value": "https://telos-testnet.eosio.online"
}]
'

Reply:
done!